assets/js/src/core/components/grid/grid-cell/grid-row.tsx

```diff
@@ -16,48 +16,56 @@ import React, { useMemo } from 'react'
 import { GridCell } from './grid-cell'
 import { type GridContextProviderProps } from '../grid-context'
 import { type GridProps } from '@Pimcore/types/components/types'
+import { Dropdown, type DropdownMenuProps } from '@Pimcore/components/dropdown/dropdown'
 
 export interface GridRowProps {
   row: Row<any>
   modifiedCells: string
   isSelected?: boolean
   tableElement: GridContextProviderProps['table']
   columns: GridProps['columns']
+  contextMenuItems?: DropdownMenuProps['items']
 }
 
-const GridRow = ({ row, isSelected, modifiedCells, ...props }: GridRowProps): React.JSX.Element => {
+const GridRow = ({ row, isSelected, modifiedCells, contextMenuItems = [], ...props }: GridRowProps): React.JSX.Element => {
   const memoModifiedCells = useMemo(() => { return JSON.parse(modifiedCells) }, [modifiedCells])
 
   return useMemo(() => {
     return (
-      <tr
-        className={ ['ant-table-row', row.getIsSelected() ? 'ant-table-row-selected' : ''].join(' ') }
+      <Dropdown
```

Comment:
Can we add the dropdown only when items are defined? Currently we have the situation that in grids without dropdown items (e.g. properties or metadata) the right click of the default context menu of the browser is deactivated. It would be an advantage if this default context menu still works in that case.

Reply:
changed with: https://github.com/pimcore/studio-ui-bundle/pull/847/commits/87e11c17c6cc0997366ccce343242a4d80e2d962